src/game_config_manager.cpp

```diff
@@ -129,12 +129,13 @@ bool map_includes(const preproc_map& general, const preproc_map& special)
 }
 } // end anonymous namespace
 
+#define LOG_GC_DEFINES FORCE_LOG_TO(out, log_config)
```

Comment:
What does this line do?

Reply:
That line doesn't do anything - it's a relic of an alternate adjustment I considered making to the code in this function.